stable/jenkins/values.yaml

```diff
@@ -360,8 +361,10 @@ NetworkPolicy:
 
 ## Install Default RBAC roles and bindings
 rbac:
-  install: false
+  install: true
```

Comment:
best practices say this field should be named `create`. maybe we can take advantage here by leaving support for `install` while adding support for `create`?
or would that be too much of a hassle?
for example:
```
{{ if or .Values.rbac.install .Values.rbac.create }}
```

wdyt?

Reply:
Technically it's possible would just look strange in the documentation having two fields with the same meaning. What would be you desired deprecation period? Same as for `.Values.rbac.roleKind` and `.Values.rbac.roleRef`? So removal before 1.0 release.

Reply:
i would remove `install` from values.yaml and from README, and add a deprecation message to NOTES.
this is also good imo because if someone has rbac disabled, this change will be a problem. we have been getting some complaints about breaking change, and i think this approach will prove to be better then simply breaking.
and yes, deprecation with 1.0.

Reply:
change is implemented. now it's called `Values.rbac.create`